.github/workflows/publish.yml

```
@@ -44,5 +45,17 @@ jobs:
           BUILD_DATE=${{ env.build_date }}
           VCS_REF=${{ env.vcs_ref }}
         cache-from: type=gha
-        cache-to: type=gha,mode=max  
-    
+        cache-to: type=gha,mode=max
+
+    - name: Build the stable OCR-D Controller image and deploy to GitHub Container Repository
+      uses: docker/build-push-action@v4
+      with:
+        context: .
+        push: true
+        tags: ${{ env.REGISTRY_PATH }}/ocrd_controller:stable
```

Perhaps we can even add some release tag here:
```suggestion
        tags: ${{ env.REGISTRY_PATH }}/ocrd_controller:stable,${{ env.REGISTRY_PATH }}/ocrd_controller:${{ env.vcs_tag }}
```

(Will be just `HEAD` if there is no tag for it.)

---

The question here was if saving the tags will cause problems on GitHub. I assumed that the stable tag simply overrides the stable image version.

---

It does. This is just to make sure we **also** produce some persistent images ourselves from time to time (viz. whenever we decide to make a release).

---

Why didn't you commit my suggestions?

---

oh I thought I had already done that. Was not intentional. I'll change that directly.

---

Added your suggest changes with seperate pr #22